Externals.cfg

```diff
@@ -51,15 +51,23 @@ required = True
 
 [blom]
 protocol = git
+<<<<<<< HEAD
 tag = 1d71682961d242493a7cb0416b60fe60af7fbc27
+=======
+tag = v1.5.0
+>>>>>>> feature/averaging_for_partitions
 repo_url = https://github.com/NorESMhub/BLOM
 local_path = components/blom
 externals = Externals_BLOM.cfg
 required = True
 
 [cam]
 protocol = git
+<<<<<<< HEAD
```

Comment:
Try the merge again?

Reply:
Sorry for that. Just cleaned up the merge and tried again.